README.md

```diff
@@ -78,7 +78,7 @@ Read more in [Why run MEV-Boost?](https://writings.flashbots.net/why-run-mevboos
 
 The most common setup is to install MEV-Boost on the same machine as the beacon client. Multiple beacon-clients can use a single MEV-Boost instance. The default port is 18550.
 
-See also [Rémy Roy's guide](https://github.com/remyroy/ethstaker/blob/main/prepare-for-the-merge.md#installing-mev-boost) for comprehensive instructions on installing, configuring and running MEV-Boost.
+See also [Rémy Roy's guide](https://github.com/eth-educators/ethstaker-guides/blob/05401128c229b72467deee0ed3951701687336a6/docs/prepare-for-the-merge.md#installing-mev-boost) for comprehensive instructions on installing, configuring and running MEV-Boost.
```

Comment:
```suggestion
See also [Rémy Roy's guide](https://github.com/eth-educators/ethstaker-guides/blob/main/docs/prepare-for-the-merge.md#installing-mev-boost) for comprehensive instructions on installing, configuring and running MEV-Boost.
```

better tag the main branch, not a specific commit so users will always see the newest version